vuepress/docs/next/tutorials/samples/customer-portal.md

```diff
@@ -0,0 +1,166 @@
+The administrator begins at Jira Service Management to create users, projects, define the organization, and configure the service ticket system.
+
+Users who need access to the Customer Portal beyond subscription and project information, must have a Jira Service Management account.
+
+1. Go to Customers to add organizations and projects. Once added, click on the name of the organization to get the ID, needed later, from the URL. \
+ e.g. example.com/jira/servicedesk/projects/ECS/organization/3 → the organization ID is “3”
+
+### Entando Identity Management System 
+Logging into the Entando Identity Management System, you will see the landing page shown below. Like the App Builder, the left navigation bar is your guide for managing users, groups, and most importantly, roles. Using the RBAC model, define what access users have by the roles and groups they are assigned. Some important guidelines are noted below. 
+
+![Entando Identity Management System](./images/cp-idmanagement-main.png)
+
+* **Define the Realm Setting**. \
+The `Realm` is a set of users, credentials, roles, and groups. A user belongs to and logs into a `Realm`. 
+
+* **Create Roles**\
+You can use the default roles by clicking on `Client Roles` and choosing `entandodemo-customerportal-server`. Access for each role is defined as follows:
+    * `cp-customer` -  assigned directly to specific projects for a single customer
+    * `cp-partner` - assigned directly to specific projects for multiple customers
+    * `cp-support` -  read only view of all customer projects
+    * `cp-admin` - admin access for the Customer Portal
+
+* **Create New Users**: \
+     1. From the navigation bar, go to Users. Click `Add User` at right.
+     2. Complete the form as needed but note the requirements for the following fields:\
+       `Username`: a unique name\
+       `Email`: must use the same address used in Jira\
+       `User Enabled` → On\
+     `Save`
+
+     3. Send an email to the user to activate their account and set a new password: \
+     Go to `Credentials` header. \
+     Under `Credential Reset`, in the `Reset Actions` → `Update Password`.  \
+     Click `Send Email`. \
+
+     4. Go to Role Mapping. \
+     Select the appropriate roles from `Available Roles` and click `Add Selected` to assign. 
+     * Choose `entandodemo-customerportal-server` from the `Client Roles` pull-down menu to use the default roles. 
+     * A full administrative user will need `realm-admin` role under `realm-management` Client Roles in order to manage users in the Portal. 
+     * Check the `Effective Roles` column on right to ensure the correct roles have been assigned.
+
+![Entando ID Management Role Mapping](./images/cp-identity-userrole.png)
+
+  5. Under `Groups`, assign roles to groups as needed. Roles are additive.
+
+::: tip Notes:
+* The built-in mapper for email must be enabled on the server client so that user accounts can be retrieved from Jira and tickets created use that account information.
```

Comment:
These need to be front and center right after the install since they are the first steps required to get the portal working. Let's move the tips to their own section at the top of Configuration, maybe as Initial Configuration Steps?  What you have is fine for content and I opened ENDOC-403 for the next round of changes.